flybywiresim/aircraft
# fix(a380x/fws): Various FWS fixes

## Summary of Changes


- Fix typo, missing L: when reading A32NX_LGCIU_1_L_GEAR_COMPRESSED
- No T/O flaps warning when in CONF 3 (allowable T/O config on the 380)
- Fix overspeed warnings for clean conf and conf 1
- Don't trigger ADR fault warnings, when securing aircraft after flight (i.e. setting IR mode sel. to OFF)

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


- Perform CONF 3 take-off, shouldn't trigger any ECAM warning
- Verify that overspeed warnings work as expected (they only take the actual config in consideration, which is not neccessarily what the flaps lever shows)
- Turn off ADR of your choice in different phases of flight, shouldn't trigger a warning on ground before engine start and after engine shutdown


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
